thriftbp/prometheus.go

```diff
@@ -0,0 +1,38 @@
+package thriftbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	thriftLabels = []string{
+		"thrift_service",
+		"thrift_method",
+		"thrift_success",
+		"thrift_exception_type",
+		"thrift_baseplate_status",
+		"thrift_baseplate_status_code",
```

Comment:
nit: the same string literals are used both here and in `PrometheusMetricMiddleware`, although this is unlikely to happen but it could still potentially happen that in the future someone changed the string in one place and forgot to change it in the other place and that will cause issues, so I'd prefer to define them as constants to avoid that issue.

Reply:
good idea. let me fix that.